### ISSUE TITLE
Don't see any linters

### ISSUE THREAD

**Issue by asaadmahmoodspin** [SublimeLinter/SublimeLinter3#391]
I have jshint and eslint howevevr I don't see any errors in my JS files.
![screen shot 2016-01-08 at 1 21 53 am](https://cloud.githubusercontent.com/assets/11034289/12181838/5e4d223a-b5a6-11e5-8c28-038a509ecb29.png)

Here's my console
```
startup, version: 3062 osx x64 channel: dev
executable: /Applications/Sublime Text.app/Contents/MacOS/Sublime Text
working dir: /
packages path: /Users/asaadmahmood/Library/Application Support/Sublime Text 3/Packages
state path: /Users/asaadmahmood/Library/Application Support/Sublime Text 3/Local
hardware concurrency: 8
zip path: /Applications/Sublime Text.app/Contents/MacOS/Packages
zip path: /Users/asaadmahmood/Library/Application Support/Sublime Text 3/Installed Packages
found 7 files for base name Default.sublime-keymap
found 1 files for base name Default.sublime-mousemap
found 6 files for base name Main.sublime-menu
loading bindings
loading pointer bindings
found 1 files for base name Default.sublime-theme
theme loaded
app ready
wrote startup cache, added files: 1 orphaned files: 0 total files: 194 cache hits: 193
pre session restore time: 0.700503
using gamma: 1.9 (err: 99.6544)
startup time: 0.740455
first paint time: 0.76327
scan: /Users/asaadmahmood/Git/projects/go/src/github.com/mattermost/platform/web/react/node_modules/highlight.js/styles has been seen before, skipping (using inode) previous path: /Users/asaadmahmood/Git/projects/go/src/github.com/mattermost/platform/web/static/css/highlight inode: 76223614
error: unable to open crawldb (~/Library/Application Support/Sublime Text 3/Index), indexing is unavailable
launching: /Applications/Sublime Text.app/Contents/MacOS/plugin_host
reloading plugin Default.block
reloading plugin Default.comment
reloading plugin Default.copy_path
reloading plugin Default.delete_word
reloading plugin Default.detect_indentation
reloading plugin Default.duplicate_line
reloading plugin Default.echo
reloading plugin Default.exec
reloading plugin Default.fold
reloading plugin Default.font
reloading plugin Default.goto_line
reloading plugin Default.history_list
reloading plugin Default.indentation
reloading plugin Default.kill_ring
reloading plugin Default.mark
reloading plugin Default.new_templates
reloading plugin Default.open_file_settings
reloading plugin Default.open_in_browser
reloading plugin Default.pane
reloading plugin Default.paragraph
reloading plugin Default.paste_from_history
reloading plugin Default.save_on_focus_lost
reloading plugin Default.scroll
reloading plugin Default.set_unsaved_view_name
reloading plugin Default.side_bar
reloading plugin Default.sort
reloading plugin Default.swap_line
reloading plugin Default.switch_file
reloading plugin Default.symbol
reloading plugin Default.transform
reloading plugin Default.transpose
reloading plugin Default.trim_trailing_white_space
reloading plugin CSS.css_completions
reloading plugin Diff.diff
reloading plugin HTML.encode_html_entities
reloading plugin HTML.html_completions
reloading plugin 0_package_control_loader.00-package_control
reloading plugin 0_package_control_loader.02-bz2
reloading plugin All Autocomplete.all_views_completions
reloading plugin AutoFileName.autofilename
reloading plugin AutoFileName.getimageinfo
reloading plugin Emmet.emmet-plugin
reloading plugin Package Control.1_reloader
reloading plugin Package Control.2_bootstrap
reloading plugin Package Control.Package Control
reloading plugin SideBarEnhancements.SideBar
reloading plugin SideBarEnhancements.SideBarAPI
reloading plugin SideBarEnhancements.SideBarDefaultDisable
reloading plugin SublimeLinter-jshint.linter
loaded 2383 snippets
SublimeLinter: jshint linter loaded 
reloading plugin SublimeLinter.commands
reloading plugin SublimeLinter.sublimelinter
reloading plugin SublimeLinter-contrib-eslint.linter
SublimeLinter: eslint linter loaded 
plugins loaded
SublimeLinter: debug mode: off 
SublimeLinter: jshint activated: /usr/local/Cellar/node/0.10.32/bin/jshint 
found 6 files for base name Main.sublime-menu
Emmet: No need to update PyV8
Package Control: Skipping automatic upgrade, last run at 2016-01-08 01:16:39, next run at 2016-01-08 02:16:39 or after
```

And here's my user settings file.

```
{
    "user": {
        "@disable": true,
        "debug": false,
        "delay": 0.25,
        "error_color": "D02000",
        "gutter_theme": "Packages/SublimeLinter/gutter-themes/Default/Default.gutter-theme",
        "gutter_theme_excludes": [],
        "lint_mode": "background",
        "linters": {
            "eslint": {
                "@disable": false,
                "args": [],
                "excludes": [],
                "ignore_match": ""
            },
            "jshint": {
                "@disable": false,
                "args": [],
                "excludes": []
            }
        },
        "mark_style": "outline",
        "no_column_highlights_line": false,
        "passive_warnings": false,
        "paths": {
            "linux": [],
            "osx": [
                "/usr/local/Cellar/node/0.10.32/bin"
            ],
            "windows": []
        },
        "python_paths": {
            "linux": [],
            "osx": [],
            "windows": []
        },
        "rc_search_limit": 3,
        "shell_timeout": 10,
        "show_errors_on_save": false,
        "show_marks_in_minimap": true,
        "sublimelinter_disable": [],
        "syntax_map": {
            "html (django)": "html",
            "html (rails)": "html",
            "html 5": "html",
            "javascript (babel)": "javascript",
            "php": "html"
        },
        "warning_color": "DDB700",
        "wrap_find": true
    }
}
```

**Comment by grotewold** [comment]
What syntax is your view using?

**Comment by asaadmahmoodspin** [comment]
Javascript.

**Comment by asaadmahmoodspin** [comment]
![screen shot 2016-01-08 at 9 45 44 am](https://cloud.githubusercontent.com/assets/11034289/12190720/ccb17be4-b5ed-11e5-96f0-ed1f3ae17eb3.png)

**Comment by asaadmahmoodspin** [comment]
Reinstalled sublime and added the package again and it started working. Phew.

**Comment by grotewold** [comment]
Glad to hear you found your problem! :cheers:

> On Jan 8 2016, at 1:58 am, Asaad Mahmood &lt;notifications@github.com&gt;
wrote:  

>

> Closed [#391](https://github.com/SublimeLinter/SublimeLinter3/issues/391).

>

> —  
Reply to this email directly or [view it on GitHub](https://github.com/Sublime
Linter/SublimeLinter3/issues/391#event-509111553).![](https://github.com/notif
ications/beacon/ABWx8jLlvbkE0O_3icDzjz4TcNT_l87Sks5pX1V2gaJpZM4HArY6.gif)
